saleor/integrations/feeds.py

```diff
@@ -0,0 +1,212 @@
+            return 'in stock'
+        else:
+            return 'out of stock'
+
+    def item_shipping(self, item):
+        """
+        You can set one shipping cost in feed settings or precise shipping cost for every item.
+        Read more:
+        https://support.google.com/merchants/answer/7050921
+        """
+        return None
+
+    def item_google_product_category(self, item):
+        """
+        To have your categories accepted, please use names accepted by Google or
+        write custom function which maps your category names into to Google codes.
+        Read more:
+        https://support.google.com/merchants/answer/6324436
+        """
+        category = item.product.get_first_category()
+        if category:
+            category_path = self.get_full_category_name_path(category)
+            return category_path
+        else:
+            return ''
+
+    def item_price(self, item):
+        price = item.get_price_per_item(discounts=None)
+        return '%s %s' % (price.gross, price.currency)
+
+    def item_sale_price(self, item):
+        sale_price = item.get_price_per_item(discounts=self.discounts)
+        return '%s %s' % (sale_price.gross, sale_price.currency)
+
+    def item_attributes(self, item):
+        product_data = {
+            'id': self.item_id(item),
+            'title': self.item_title(item),
+            'description': self.item_description(item),
+            'condition': self.item_condition(item),
+            'mpn': self.item_mpn(item),
+            'item_group_id': self.item_group_id(item),
+            'availability': self.item_availability(item),
+            'google_product_category': self.item_google_product_category(item),
+            'link': self.item_link(item),
+            'tax': self.item_tax(item),
+            'shipping': self.item_shipping(item)
+        }
+
+        image_link = self.item_image_link(item)
+        if image_link:
+            product_data['image_link'] = image_link
+
+        price = self.item_price(item)
+        product_data['price'] = price
+        sale_price = self.item_sale_price(item)
+        if sale_price != price:
+            product_data['sale_price'] = sale_price
+
+        tax = self.item_tax(item)
+        if tax:
+            product_data['tax'] = tax
+
+        shipping = self.item_shipping(item)
+        if shipping:
+            product_data['shipping'] = shipping
+
+        brand = self.item_brand(item)
+        if brand:
+            product_data['brand'] = brand
+
+        return product_data
+
+
+class SaleorFeed(GoogleProductFeed):
+    """
+    Example of using GoogleProductFeed.
+    """
+    file_path = path.join(settings.INTEGRATIONS_DIR, 'saleor-feed.csv.gz')
+    file_url = settings.MEDIA_URL + file_path
+
+    def item_shipping(self, item):
+        """Flat shipping price"""
+        price = Price(5, currency=settings.DEFAULT_CURRENCY)
+        return 'US:::%s %s' % (price.gross, price.currency)
```

Comment:
Where did price this come from? This is surely not how we calculate shipping costs?